iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientComponentTests.java

```diff
@@ -193,13 +192,13 @@ public void invokeComponentLevelCommand() throws IOException, InterruptedExcepti
 
         // assert
         assertEquals(deviceSuccessResponseStatus, responseWithNoPayload.getStatus());
-        assertEquals("\"\"", responseWithNoPayload.getPayload());
+        assertEquals("", responseWithNoPayload.getPayloadAsJsonString().replace("{}", ""));
```

Comment:
This feels a bit odd. Why are you changing the **actual** result (replacing "{}" with "") when you could just change the **expected** result?